backend/app/controllers/spree/admin/stores_controller.rb

```diff
@@ -6,7 +6,7 @@ class StoresController < Spree::Admin::BaseController
       before_action :set_default_locale, only: :new
       before_action :normalize_supported_currencies, only: [:create, :update]
       before_action :set_default_country_id, only: :new
-      before_action :load_all_countries, only: [:new, :edit, :update]
+      before_action :load_all_countries, only: [:new, :edit, :update, :create]
```

Comment:
@MatthewKennedy can you extract this change to a separate PR? thanks!

Reply:
Sure thats not a problem.